portfolio/src/main/webapp/style.css

```diff
@@ -1,17 +1,57 @@
+*{
+    background-color: #264653;
+    font-family: 'Raleway', sans-serif;
+    color: white;
+}
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
-  font-family: Arial, Helvetica, sans-serif;
+  
+}
+
+#content h1{
+    color:#2a9d8f;
+}
+
+button{
+    padding: 16px 32px;
+    background-color: #264653; 
+    color: #2a9d8f; 
+    border: 2px solid #2a9d8f;
+}
+
+button:hover{
+    background-color: #2a9d8f;
+    color:#264653;
 }
 
 .avatar{
     width: 250px;
     height: 250px;
 }
 
+#About h2{
```

Comment:
(Optional) You could consider just having a separate `#About` and putting the color under it since they're the same color.